From 6012e2f062edc2b5d968f68f0adc2cd96671f728 Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Fri, 13 Jan 2012 16:10:55 +0800
Subject: [PATCH 190/280] ENGR00171978 vpu: remove usleep workaround in vpu
 lib

usleep(10) was added as workaround for multi-instances task competition.
Remove it now since it affected performance a little due to task schedule
for one standalone instance decoding. The application can handle
whether need to delay some time in multi-instances.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 vpu/vpu_lib.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/vpu/vpu_lib.c b/vpu/vpu_lib.c
index d191f46..327dae3 100644
--- a/vpu/vpu_lib.c
+++ b/vpu/vpu_lib.c
@@ -4308,9 +4308,6 @@ RetCode vpu_DecGetOutputInfo(DecHandle handle, DecOutputInfo * info)
 	*ppendingInst = 0;
 	UnlockVpu(vpu_semap);
 
-	/* Workaround for multi-instances competition */
-	usleep(10);
-
 	return RETCODE_SUCCESS;
 }
 
-- 
1.8.0

